refactor: Remove deprecation related to cookies (#9406)
codeigniter4/CodeIgniter4: user_guide_src/source/libraries/cookies.rst

Dispatching Cookies in Store
============================

> .. important:: This method is deprecated since 4.1.6 and removed in 4.6.0.

Reviewer: ```suggestion
.. important:: This method was deprecated as of version 4.1.6 and was removed in 4.6.0.
```

    .. php:method:: withNeverExpiring()

>         .. important:: This method is deprecated since 4.2.6 and removed in 4.6.0

Reviewer: ```suggestion
        .. important:: This method was deprecated as of version 4.2.6 and was removed in 4.6.0.
```

    .. php:method:: dispatch(): void

>         .. important:: This method is deprecated since 4.1.6 and removed in 4.6.0.

Reviewer: ```suggestion
        .. important:: This method was deprecated as of version 4.1.6 and was removed in 4.6.0.
```
